### PR TITLE
minor changes + forbidden winds sound effect

### PR DESCRIPTION
**Pull request by sbubby**
Replaces the sound effect for obtaining forbidden winds.
Replaces the shrine of combat skeleton archers with hoplites.
Added to the description of the Hermes vow.
Added Crafting recipes for desert monoliths.

### CODE REVIEW

File: Content/Tiles/Desert/DesertMonolith.cs [hunk 1]
```diff
@@ -113,6 +113,15 @@ internal class DesertMonolithItem : QuickTileItem
 		public override string Texture => AssetDirectory.DesertTile + Name;
 
 		public DesertMonolithItem() : base("Desert Monolith", "Places a section of desert monolith", "DesertMonolith") { }
+		public override void AddRecipes()
```
Comment (ScalarVector1) [inline Content/Tiles/Desert/DesertMonolith.cs:116]: missing linebreak

File: Content/Tiles/Desert/DesertMonolith.cs [hunk 2]
```diff
@@ -129,6 +138,15 @@ internal class DesertMonolithFlippedItem : QuickTileItem
 	{
 		public override string Texture => AssetDirectory.DesertTile + Name;
 
-		public DesertMonolithFlippedItem() : base("Flipped Desert Monolith", "Places a section of flipped desert monolith", "DesertMonolithFlipped") { }
+		public DesertMonolithFlippedItem() : base("Flipped Desert Monolith", "Places a section of flipped desert monolith\n'Even the text is flipped!'", "DesertMonolithFlipped") { }
+		public override void AddRecipes()
```
Comment (ScalarVector1) [inline Content/Tiles/Desert/DesertMonolith.cs:142]: missing linebreak

**Review by ScalarVector1 — CHANGES_REQUESTED** [review]
Please fix formatting and update PATCH_NOTES.txt

**Comment by sbubby** [comment]
i wasn't sure where to put the desert monolith recipe in the patch notes

**Review by ScalarVector1 — APPROVED** [review]

**Pull request merged**